<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>0</y>
      <width>221</width>
      <height>342</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="Draw_but_3">
       <property name="text">
        <string>Отрисовать график</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear_But_3">
       <property name="text">
        <string>Очистить экран</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>X = [</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_EDIT_MIN">
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_EDIT_MAX">
         <property name="text">
          <string>1</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>a =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="a_param_input">
           <property name="text">
            <string>0.01</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>b =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="b_param_input">
           <property name="text">
            <string>5</string>
           </property>
           <property name="maxLength">
            <number>32767</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Начальное значение y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Start_Posit">
             <property name="text">
              <string>1</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Начальное значение y':</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Start_y_for_sys">
             <property name="text">
              <string>1</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Точность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Acc_Edit_3">
           <property name="text">
            <string>0.0001</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Шаг по пространству</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="StepEdit_3">
           <property name="text">
            <string>0.01</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="Test_task_button">
       <property name="text">
        <string>Пробная задача</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="main_task">
       <property name="text">
        <string>Основная задача №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="main_task_syst">
       <property name="text">
        <string>Основная задача №2(сист)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>461</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="xy1">
       <property name="text">
        <string>График (x, y)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="xy2">
       <property name="text">
        <string>График (x, y')</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="y1y2">
       <property name="text">
        <string>Фазовый портрет (y, y')</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="aux_info">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
